test/scaleTest.js

```diff
@@ -0,0 +1,36 @@
+/* global suite test assert */
+
+function targetTester (keyframeA, keyframeB, timing) {
```

Comment:
targetTester is not a very descriptive name. It should communicate what the function does succinctly.